viite-UI/src/view/ProjectEditForm.js

```diff
@@ -122,7 +122,8 @@
       var inputs = rootElement.find('input');
       var filled = true;
       for (var i = 0; i < inputs.length; i++) {
-        if (inputs[i].type === 'text' && !inputs[i].value) {
+        var isPedestrian = $('#tie')[0].value >= 70000; // roads with road number over 70000 are pedestrian roads
```

Comment:
we rather not have magic number, even though comment is nice, but it is better to initialize pedestrianroads variable/value and use that. Then incase someone decides pedestrian roads start at 70000 it is easier to fix